.github/workflows/prettier.yml

```diff
@@ -0,0 +1,21 @@
+on: [push, workflow_dispatch]
```

Comment:
I'm wondering, instead of doing it on every push, maybe we can run it based on `pull_request`:`open` and `edited` event types?
(We shouldn't enforce actions like these on all pushes on forks of our repo, if someone isn't opening PRs)

Reply:
It has too many issues with authorization.

Reply:
That's true...
Perhaps we can make it _not_ directly commit to at least the main branch, unless it's a PR?

Reply:
It will only run on forks and if it runs on every fork, so it won't push to the main until you change the contents of public/data/*.json that would be prolly only done when making PR.

Reply:
welp, I guess we can live with it for now
(Hopefully, we shift to db stored data soon...)